exercises/farewell-workflow/solution/Client/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Run Workflow file
+using Temporalio.Client;
+using Temporalio.Farewell.Workflow;
+
+// Connect to the Temporal server
+var client = await TemporalClient.ConnectAsync(new("localhost:7233") { Namespace = "default" });
+
+var workflowId = $"greeting-{Guid.NewGuid()}";
+
+// Default to "Temporal" and overriding if a valid argument is provided
+string name = (args.Length > 0 && !string.IsNullOrEmpty(args[0])) ? args[0] : "Temporal";  // Default name

```

Comment:
```suggestion
var name = args.FirstOrDefault() ?? "Temporal";  // Default name
```

This is a bit clearer (untested),. But why default to `"Temporal"` if you're just going to ask later and default again?